### DISCUSSION TITLE
How to configure "volumeMounts" for OAPServer?

### DISCUSSION THREAD

**Discussion by liwei-tmx** [apache/skywalking#11519] [Q&A] [answered]
`apiVersion: operator.skywalking.apache.org/v1alpha1
kind: OAPServer
metadata:
  name: skywalking
spec:
  version: 9.5.0
  instances: 1
  image: apache/skywalking-oap-server:9.5.0
  service:
    template:
      type: ClusterIP
  
---
apiVersion: operator.skywalking.apache.org/v1alpha1
kind: UI 
metadata:
  name: skywalking
spec:
  version: 9.5.0
  instances: 1
  image: apache/skywalking-ui:9.5.0
  OAPServerAddress: http://skywalking-oap.default:12800
  service:
    template:
      type: ClusterIP
    ingress:
      host: demo.ui.skywalking
`

**Comment by wu-sheng** [comment]
Could you share what do you do by mounted volume?

**Reply by wu-sheng** [reply]
Usually, we don't have volume for OAP.

**Reply by liwei-tmx** [reply]
Thank you for your reply! What should I do when I need to add an additional jar to the oap-server oap-libs folder?

**Reply by wu-sheng** [reply]
Are you using helm or operator?

**Reply by liwei-tmx** [reply]
[https://skywalking.apache.org/zh/2022-04-19-how-to-use-the-java-agent-injector/](url)
operator, the deployment steps refer to this blog.

**Reply by liwei-tmx** [reply]
Start an arthas diagnostic process from skywalking-ui, need copy arthas-controller to oap-server oap-libs folder.
[https://github.com/weixiang1862/skywalking-x-arthas#3-copy-arthas-controller-to-your-oap-server-oap-libs-folder](url)

**Reply by wu-sheng** [reply]
@dashanji Any suggestion?

**Reply by dashanji** [reply, marked as answer]
Hi @liwei-tmx , current `OAPServer` doesn't support configuring "volumeMounts". 

A workaround is to edit the **OAPServer Deployment** via kubectl manually and mount the volume that contains the `arthas-controller` jar.

**Reply by liwei-tmx** [reply]
thank you ,I will try.
